ios/Podfile.lock

```diff
@@ -1164,4 +1173,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: a409230d2dbff899bf9ff74097f2aa21f5a54a2a
 
-COCOAPODS: 1.14.3
+COCOAPODS: 1.15.2
```

Comment:
we cannot upgrade `cocoapods` to 1.15.2 due to a blocker on react-native dependency, please use bundle to execute the `pod install`

```shell
bundle install
cd ios
bundle exec pod install
```

Reply:
Solved with the following commit → https://github.com/pagopa/io-app/pull/5954/commits/55a684a67d09c7b4a9a9d539e0afd48dc365d4a6